src/commands/locations/modes/delete.ts

```diff
@@ -0,0 +1,50 @@
+import { type ArgumentsCamelCase, type Argv, type CommandModule } from 'yargs'
+
+import { apiCommand, apiCommandBuilder, apiDocsURL, type APICommandFlags } from '../../../lib/command/api-command.js'
+import { chooseMode } from '../../../lib/command/util/modes-choose.js'
+
+
+export type CommandArgs =
+	& APICommandFlags
+	& {
+		location?: string
+		id?: string
+	}
+
+const command = 'locations:modes:delete [id]'
+
+const describe = 'delete a mode'
+
+const builder = (yargs: Argv): Argv<CommandArgs> =>
+	apiCommandBuilder(yargs)
+		.positional('id', { describe: 'mode id', type: 'string' })
+		.option('location', { alias: 'l', describe: 'a specific location to create the mode in', type: 'string' })
```

Comment:
"to delete the mode from"

Reply:
Oops. Ugh, this is another one of those that's super weird to write correctly. It's not really "to delete the mode from" either though, it's just a filter for when the user doesn't specify a node. I've changed it to the generic "to query" we've been using everywhere else.